src/main/webapp/user_profile.html

```diff
@@ -21,49 +21,94 @@ <h1 id="title">My Tasks</h1>
+            <button class="help-button" id="offer-help-button" onclick="showOfferHelp()">Offer help</button>
+            <i class="fa fa-plus-circle fa-3x" id="create" onclick="showModal()"></i>
         </div>
     </div>
     <br/>
     <div id="task-list">
-        <table id="need-help">
+        <table id="need-help"></table>
+        <table id="offer-help">
             <tr>
                 <th>Task Overview</th>
-                <th>Assignee</th>
-                <th>Status</th>
-                <th>Edit</th>
-                <th>Delete</th>
+                <th>Neighbor</th>
+                <th>Reward</th>
+                <th>Mark as complete</th>
+                <th>Abandon</th>
             </tr>
             <tr>
                 <td>Collect the garbage on the drive</td>
                 <td>Leonard</td>
-                <td>OPEN</td>
-                <td><button class="edit-task"><i class="fa fa-edit"></i></button></td>
-				<td><button class="delete-task" onclick="deleteRow(this)">
-                    <i class="fa fa-trash-o"></i>
+                <td>100 pts</td>
+				<td><button class="complete-task" onclick="completeTask(this)">
+                    <i class="fa fa-check"></i>
+                </button></td>
+                <td><button class="abandon-task" onclick="completeTask(this)">
+                    <i class="fa fa-times"></i>
                 </button></td>
             </tr>
             <tr>
                 <td>Mown the grass in the backyard</td>
                 <td>Jack</td>
-                <td>OPEN</td>
-                <td><button class="edit-task"><i class="fa fa-edit"></i></button></td>
-                <td><button class="delete-task" onclick="deleteRow(this)">
-                    <i class="fa fa-trash-o"></i>
+                <td>150 pts</td>
+                <td><button class="complete-task" onclick="completeTask(this)">
+                    <i class="fa fa-check"></i>
+                </button></td>
+                <td><button class="abandon-task" onclick="completeTask(this)">
+                    <i class="fa fa-times"></i>
                 </button></td>
             </tr>
             <tr>
                 <td>Help me grab some bread and milk from supermarket</td>
                 <td>Jason</td>
-                <td>IN PROGRESS</td>
-                <td><button class="edit-task"><i class="fa fa-edit"></i></button></td>
-                <td><button class="delete-task" onclick="deleteRow(this)">
-                    <i class="fa fa-trash-o"></i>
+                <td>250 pts</td>
+                <td><button class="complete-task" onclick="completeTask(this)">
+                    <i class="fa fa-check"></i>
+                </button></td>
+                <td><button class="abandon-task" onclick="completeTask(this)">
+                    <i class="fa fa-times"></i>
                 </button></td>
             </tr>
         </table>
     </div>
+    <div id="createTaskModal">
```

Comment:
There might be a way of consolidating this into the same form, I'm not sure if it'll even be better or not... I'll look into it! It just seems like they're almost exactly the same minus the http method request, so it might be something to think about :)

Reply:
Yep. This kind of code replication really annoys me but I haven't came up with a good idea. I may leave them for the prototype but think about how to fix this in the MVP.